debug-ui/package.json

```diff
@@ -3,9 +3,11 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "preinstall": "cd ../ && yarn build",
+    "preinstall": "cd ../ && yarn install --frozen-lockfile && yarn build",
     "dev": "next dev",
     "build": "next build",
+    "format": "prettier -c app/ lib/ scripts/",
+    "format:fix": "prettier -w app/ lib/ scripts/",
```

Comment:
is there a reason why you are not simply using the lint and lint:fix commands?

Reply:
ah it looks like prettier is not used in there... might as well combine the two though. or does this affect some CI stuff? I would prefer lint and lint:fix to handle formatting as well and just have the single set of commands.

Reply:
could combine into a higher level taks "check" & "fix". would prefer them separate for ci